lib/loader.js

```diff
@@ -101,4 +89,40 @@ function sassLoader(content) {
     });
 }
 
+/**
+ * Verifies that the implementation and version of Sass is supported by this loader.
+ *
+ * @param {string} info
+ */
+function validateSassVersion(info) {
+    const parsed = /^([^\t]+)\t([^\t]+)/.exec(info);
+
+    if (!parsed) {
+        throw new Error("Unknown Sass implementation \"" + info + "\".");
+    }
+
+    const implementation = parsed[1];
+    const version = parsed[2];
+    const components = /^(\d+)\.(\d+)/.exec(version);
+
+    if (!components) {
+        throw new Error("Invalid Sass version \"" + version + "\".");
+    }
+
+    const major = Number(components[1]);
+    const minor = Number(components[2]);
+
+    if (implementation === "dart-sass") {
+        if (major < 1 || major > 1 || minor < 3) {
+            throw new Error("Dart Sass version " + version + " is incompatible with ^1.3.0.");
+        }
+    } else if (implementation === "node-sass") {
+        if (major < 4 || major > 4) {
```

Comment:
Can you provide information why `node-sass > 4` is not compatibility?

Reply:
Node Sass 5 hasn't been released yet. According to semver, it may introduce arbitrary breaking changes, so it's not safe to eagerly declare compatibility with it.